engine/admin/start-containers-automatically.md

```diff
@@ -0,0 +1,57 @@
+---
+description: How to generate scripts for upstart, systemd, etc.
+keywords: systemd, upstart, supervisor, docker, documentation,  host integration
+redirect_from:
+- /engine/articles/host_integration/
+- /engine/admin/host_integration/
+title: Start containers automatically
+---
+
+Docker provides [restart policies](/engine/reference/run.md#restart-policies---restart)
+to control whether your containers start automatically when they exit, or when
+Docker restarts. Restart policies ensure that linked containers are started in
+the correct order. Docker recommends that you use restart policies, and avoid
+using process managers to start containers.
+
+## Use a restart policy
+
+To configure the restart policy for a container, use the `--restart` flag
+when using the `docker run` command. The value of the `--restart` flag can be
+any of the following:
+
+| Flag             | Description                                                                                     |
+|:-----------------|:------------------------------------------------------------------------------------------------|
+| `no`             | Do not automatically restart the container. (the default)                                       |
+| `on-failure`     | Restart the container if it exits due to an error.                                              |
```

Comment:
Should we mention "non-zero exit code of the container's process"? (to be more specific about "error")?